src/Workspace.elm

```diff
@@ -116,7 +116,7 @@ update env msg ({ workspaceItems } as model) =
             in
             ( { model | workspaceItems = nextWorkspaceItems }
             , cmd
-            , openDefinitionsFocusToOutMsg nextWorkspaceItems
+            , None
```

Comment:
(this is the `FetchItemFinished` pattern match)

We don't need to change the URL to the focused item after fetching anymore since we are now fetching what was in the URL.